src/fr/composants/bouton-radio/base.md

```diff
@@ -7,6 +7,9 @@ tags: ['radioFR', 'header']
 ---
 
 # Bouton radio <br>`<gcds-radio>`
+<gcds-error-message message="[FR] This component is deprecated as of v0.21.0"></gcds-error-message>
+
+[FR] Please see <gcds-link href="https://github.com/cds-snc/gcds-components/releases/tag/gcds-components-v0.21.0">Release v0.21.0</gcds-link> for details
```

Comment:
Could we get translations for these please?
> This component is deprecated as of v0.21.0

and

> Please see Release v0.21.0 for details

Reply:
Ce composant est obsolète à partir de la version 0.21.0.
Consultez les notes de publication v0.21.0 pour plus d'information.